Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "traceme"
+version = "0.1.0"
+authors = ["Edd Barrett <vext01@gmail.com>"]
+
+[dependencies]
+libc = "0.2.33"
+clippy = {version = "*", optional = true}
+
+[build-dependencies]
+gcc = "0.3.54"
+
+[features]
+default = []
```

Comment:
What does this do?

Reply:
Again, taken from the clippy instructions.

Reply:
OK, we can kill this then.